validate.go

```diff
@@ -163,6 +163,8 @@ func checkLinux(spec rspec.Linux, rootfs string) {
 	for index := 0; index < len(spec.Namespaces); index++ {
 		if !namespaceValid(spec.Namespaces[index]) {
 			logrus.Fatalf("Namespace %s is invalid.", spec.Namespaces[index])
+		} else if spec.Namespaces[index].Type == rspec.UTSNamespace && hostname != "" {
```

Comment:
```
 On Linux, you can only set this if your bundle creates a new UTS namespace.
```
So host name could be empty even a config file has a UTS namespace.
We should get an error when hostname is empty and no UTS namespace is created